netbox/resource_netbox_ip_address.go

```diff
@@ -103,7 +103,7 @@ func resourceNetboxIPAddressRead(d *schema.ResourceData, m interface{}) error {
 	}
 
 	if res.GetPayload().AssignedObject != nil {
-		d.Set("interface_id", res.GetPayload().AssignedObject.ID)
+		d.Set("interface_id", res.GetPayload().AssignedObjectID)
```

Comment:
Can you do the same change here, as well?
AssignedObject is checked, but AssignedObjectID is used.
After this, we're good! :D

Reply:
done